lib/puppet/resource/type_collection.rb

```diff
@@ -187,7 +187,7 @@ def version
       if environment.config_version.nil? || environment.config_version == ""
         @version = Time.now.to_i
       else
-        @version = Puppet::Util::Execution.execute([environment.config_version]).strip
+        @version = Puppet::Util::Execution.execute(environment.config_version).strip
```

Comment:
Will this break puppet not running under puppetserver? Will this cause the `config_version` to be executed by the shell instead of `execve`?